core/keyserver/epochs_test.go

```diff
@@ -38,47 +36,32 @@ const (
 	domainID = "domain"
 )
 
-func updates(t *testing.T, start, end int) []*pb.EntryUpdate {
-	if start > end {
-		t.Fatalf("start=%v > end=%v", start, end)
-	}
-	kvs := make([]*pb.EntryUpdate, 0, end-start)
+func genMutations(start, end int) []*pb.Entry {
+	mutations := make([]*pb.Entry, 0, end-start)
```

Comment:
Happy to just let this crash if `start` > `end`?

Reply:
The meaning has indeed changed.  
`ReadPage` used to return the interval `(start, end]`, which is a bit odd, but served our purposes when we were using a high water mark.  Now it returns `[start, start+pageSize]`. As a result, we need to add 1 to fetch the next page.  Is there a better way of doing this?